src/sql/azdata.proposed.d.ts

```diff
@@ -1525,4 +1525,12 @@ declare module 'azdata' {
 		 */
 		link: LinkArea;
 	}
+
+	export interface ContainerBuilder<TComponent extends Component, TLayout, TItemLayout, TPropertyBag extends ComponentProperties> extends ComponentBuilder<TComponent, TPropertyBag> {
+		withProps(properties: TPropertyBag): ContainerBuilder<TComponent, TLayout, TItemLayout, TPropertyBag>;
+	}
+
+	export interface ComponentProperties {
+		ariaLive?: string;
```

Comment:
As mentioned in the bottom, given that most components don't (and likely aren't) going to implement this it would be preferable to just add this to the specific containers that need it. We can always move it into component later if we start to see a need to have it be more generic.